src/Mod/TechDraw/Gui/QGISVGTemplate.cpp

```diff
@@ -240,26 +265,13 @@ void QGISVGTemplate::createClickHandles()
         return;
     }
 
-    QString templateFilename(QString::fromUtf8(svgTemplate->PageResult.getValue()));
-
-    if (templateFilename.isEmpty()) {
-        return;
-    }
-
-    QFile file(templateFilename);
-    if (!file.open(QIODevice::ReadOnly)) {
-        Base::Console().Error(
-            "QGISVGTemplate::createClickHandles - error opening template file %s\n",
-            svgTemplate->PageResult.getValue());
-        return;
-    }
-
+    QByteArray svgCode = svgTemplate->processTemplate().toUtf8();
+    applyWorkaround(svgCode);
     QDomDocument templateDocument;
-    if (!templateDocument.setContent(&file)) {
+    if (!templateDocument.setContent(svgCode)) {
         Base::Console().Message("QGISVGTemplate::createClickHandles - xml loading error\n");
```

Comment:
Could this introduce a regression? This is the code already used in `QGISVGTemplate::load`, so I think it should be fine.

Reply:
@WandererFan there is no `QString` overload for `QDomDocument::setContent`, needs to be `QByteArray`

Reply:
https://doc.qt.io/qt-5/qdomdocument.html#setContent-1
bool QDomDocument::setContent(const QString &text, bool namespaceProcessing, QString *errorMsg = nullptr, int *errorLine = nullptr, int *errorColumn = nullptr)

https://doc.qt.io/qt-6/qdomdocument.html#setContent
setContent(QAnyStringView text, QDomDocument::ParseOptions options = ParseOption::Default)